DataStructures/Graph.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+
+public class Graph {
+    private boolean adjMatrix[][];
+    private int numVertices;
+
+    // Initialize the matrix
+    public Graph(int numVertices) {
+        this.numVertices = numVertices;
+        adjMatrix = new boolean[numVertices][numVertices];
+    }
+
+    // Add edges
+    public void addEdge(int i, int j) {
+        adjMatrix[i][j] = true;
+        adjMatrix[j][i] = true;
+    }
+
+    // Remove edges
+    public void removeEdge(int i, int j) {
+        adjMatrix[i][j] = false;
+        adjMatrix[j][i] = false;
+    }
+
+    public void displayGraph() {
+        for (int i = 0; i < numVertices; i++) {
+            System.out.print(i + ": ");
+            for (boolean j : adjMatrix[i]) {
+                System.out.print((j ? 1 : 0) + " ");
+            }
+            System.out.println();
+        }
+    }
+
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        Graph graph = new Graph(input.nextInt());
+        Boolean flag = true;
+        while (flag) {
+            System.out.println("Select an option from the menu below");
+            System.out.println("1. Insert an edge in the Graph");
+            System.out.println("2. Delete an edge from the Graph");
+            System.out.println("3. Exit");
+
+            int choice = input.nextInt();
+            if (choice < 1 || choice > 4) {
```

Comment:
What if user chooses to type 'a'.

Reply:
It will give an input mismatch exception. Working on it and adding try-catch blocks for exception handling.